grapher/axis/Axis.ts

```diff
@@ -29,8 +29,16 @@ interface TickPlacement {
     bounds: Bounds
     isHidden: boolean
 }
-abstract class AbstractAxis {
+
+enum Orientation {
+    vertical = "vertical",
+    horizontal = "horizontal",
+}
```

Comment:
I'm anticipating that knowing the orientation will be necessary for [Gridlines, axis labels, titles](https://www.notion.so/Gridlines-axis-labels-titles-066fcc51c1dc4a36b9593159b178cfc6).

Reply:
I removed these actually, but they are coming back (in a different form) in the next PR.